### PR TITLE
Added "slug"field to alt_names

### PR DESCRIPTION
**Pull request by illing2005**
I added a `slug` field to the `AlternativeName` model. 
If you want to use `django-cities` in an i18n application you can use the translated `Places` in urls and filter on them.

### CODE REVIEW

File: cities/admin.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 from django.contrib import admin
+from django.contrib.gis import admin as geoadmin
 from .models import *
 
-class CitiesAdmin(admin.ModelAdmin):
+class CitiesAdmin(geoadmin.OSMGeoAdmin):#admin.ModelAdmin):
```
Comment (coderholic) [inline cities/admin.py]: Why the change here?

**Comment by illing2005** [comment]
Well that's not related to the other changes. I just wanted to see a map preview on the admin page. 
You can undo this if you want.

**Comment by coderholic** [comment]
Yeah please revert the OSMGeoAdmin change and resolve conflicts and then I'll be happy to merge

**Comment by blag** [comment]
Superseded by #139.

File: cities/admin.py (hunk 1 above)
Comment (mhamzawey) [reply cities/admin.py]: This change should be propagated to make sure that we have an appropriate map in the Admin instead of Nasa's Map that has almost no data in it.

Comment (blag) [reply cities/admin.py]: @mhamzawey If you write a PR that adds a map to the different model admins I would review it and possibly merge it (if I have time).
